apps/microsoft-teams/frontend/src/hooks/useGetTeamsChannels.spec.ts

```diff
@@ -10,13 +10,12 @@ vi.mock('@contentful/react-apps-toolkit', () => ({
 
 describe('useGetTeamsChannels', () => {
   it('should return channels and accurate loading state', async () => {
-    mockSdk.cma.appActionCall.createWithResponse = vi.fn().mockReturnValueOnce({
-      response: {
-        body: JSON.stringify({
-          ok: true,
-          data: mockChannels,
-        }),
+    mockSdk.cma.appActionCall.createWithResult = vi.fn().mockReturnValueOnce({
+      result: {
+        ok: true,
+        data: mockChannels,
       },
+      status: 'processing',
```

Comment:
Hm, this is an invalid state (we cannot have a result while app action is processing) so we should change the status here to "succeeded" lest someone get confused.